app/commands/user/reputation_token/award_for_pull_request_reviewers.rb

```diff
@@ -0,0 +1,57 @@
+class User
+  class ReputationToken
+    class AwardForPullRequestReviewers
+      include Mandate
+
+      initialize_with :action, :github_username, :params
+
+      def call
+        return unless just_closed?
+
+        reviewer_usernames = reviews.map { |reviewer| reviewer[:reviewer] }.uniq
+        reviewer_usernames.delete(github_username) # Don't award reviewer reputation to the PR author
+
+        reviewers = ::User.where(github_username: reviewer_usernames)
+        reviewers.find_each do |reviewer|
+          User::ReputationToken::Create.(
+            reviewer,
+            :code_review,
+            repo: repo,
+            pr_id: pr_id,
+            pr_number: pr_number,
+            external_link: external_link
+          )
+        end
+
+        # TODO: consider what to do with missing reviewers
+        missing_reviewers = reviewer_usernames - reviewers.map(&:handle)
```

Comment:
```suggestion
        missing_reviewers = reviewer_usernames - reviewers.pluck(:handle)
```

As we don't materialise reviewers elsewhere, pluck is going to be OOMs faster.

Reply:
That makes total sense. I've also updated elsewhere